packages/http-client-java/generator/http-client-generator-test/src/main/java/com/type/property/additionalproperties/models/DifferentSpreadModelArrayRecord.java

```diff
@@ -27,7 +27,7 @@ public class DifferentSpreadModelArrayRecord implements JsonSerializable<Differe
     private final String knownProp;
 
     /*
-     * Additional properties
+     * The model spread Record<ModelForRecord[]> with the different known property type
```

Comment:
This javadoc looks a bit ugly to me... Where does it come from?

Reply:
It's due to this change. https://github.com/microsoft/typespec/pull/4671#discussion_r1794758615

Reply:
I can remove the line that sets `sdkDictType`'s `doc`, so there will be no generation diff. I also feel the javadoc is a bit ugly. What do you all think? @weidongxu-microsoft  @XiaofeiCao

Reply:
After discussing offline, we will leave it for now, and we can change if there is ask/complains later.